### ISSUE TITLE
Try to install as plugin

### ISSUE THREAD

**Issue by SolidRhino** [zdharma/zinit-configs#30]
I try to install this as a plugin but the command "zinit-configs" is not registered. How can I fix this.

I use macOS with iTerm and zsh 5.8.

**Comment by MARK-PENGUIN** [comment]
Yep, i believe this is a problem with the zplugin => zinit rename, shouldn't be too hard to fix
